addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -72,7 +72,7 @@
                 direction : ltr;
             }
             &.o_list_text {
-                word-break: break-word;
+                word-wrap: break-word;
```

Comment:
Do we actually need this rule at all ?
I may have missed something but... I get the same result by removing it altogether :thinking: 

Also, didn't test thoroughly all the uses case though...

Reply:
Actually you may be right, I don't see either any difference between removing it, or having either `word-break: break-word` or `word-wrap: break-word`, not sure in which case we might need it, but it makes sense that this behavior is the default one.

Reply:
I think we need break-word so a word with word-separating character (space/dash/...) may be split over several lines if it's too long (instead of overflowing). It's rare but for example it is often an issue reported with a long URL in the text, eg. here without and with word-break:

> ![2023-11-29-160729_1920x1080_scrot](https://github.com/odoo/odoo/assets/9977887/fa6d65d0-791d-4507-a688-f122cae25ca5)

Reply:
Thanks, then yes, I think we should keep the `word-break` or `word-wrap` right? @pparidans

Reply:
Seems ok for me @pparidans , It looks like this with the current fix (tested on safari too):

<img width="738" alt="image" src="https://github.com/odoo/odoo/assets/38792563/56986aa6-1c8c-4455-8de9-9adcc13c2f6e">